frontend/src/components/common/SimpleTable.stories.tsx

```diff
@@ -223,31 +223,30 @@ ReflectInURLWithPrefix.args = {
 
 // filter Function
 
-type SimpleTableWithFilterProps = SimpleTableProps & { matchCriteria?: string[] };
+type SimpleTableWithFilterProps = SimpleTableProps & { matchCriteria?: string[]; search?: string };
 function SimpleTableWithFilter(props: SimpleTableWithFilterProps) {
-  const { matchCriteria, ...otherProps } = props;
+  const { matchCriteria, search, ...otherProps } = props;
   const filterFunc = useFilterFunc(matchCriteria);
-  return <SimpleTable filterFunction={filterFunc} {...otherProps} />;
+  return <SimpleTable filterFunction={item => filterFunc(item, search)} {...otherProps} />;
```

Comment:
Is there an API change here with useFilterFunc?

Reply:
LGTM (I don't think there's an API break).